### PR TITLE
feat(protocol-designer): onboarding animations

### PR DESCRIPTION
**Pull request by jerader**
closes AUTH-974

# Overview

Add onboarding animations to the onboarding flow

## Test Plan and Hands on Testing

Test it out in the sandbox - i feel like there are some optimization issues with step 4 and 5 since they're the biggest files. lmk if you think its worth trying to optimize more?? I'm sort of out of ideas on what to try though without risking lowering the resolution.

## Changelog

- add webm for each page of the onboarding animation
- plug each animation into the WizardBody
- delete placeholder img

## Risk assessment

low

### CODE REVIEW

File: protocol-designer/src/pages/CreateNewProtocolWizard/WizardBody.tsx [hunk 1]
```diff
@@ -119,19 +132,25 @@ export function WizardBody(props: WizardBodyProps): JSX.Element {
...
-      <Flex>
-        <StyledImg
-          //    TODO(ja, 8/7/24): delete this and add real images!!
-          src={imgSrc ?? temporaryImg}
-          width="100%"
-          height="100%"
-        />
+      <Flex width="40%">
+        <video
+          preload="auto"
```
Comment (koji) [inline protocol-designer/src/pages/CreateNewProtocolWizard/WizardBody.tsx:157]:
```suggestion
          type="video/webm"
          preload="auto"
```

**Comment by koji** [comment]
Screenshot 2024-11-21 at 10 50 02 AM

 it is partially missing. is this an expected behavior?

**Comment by jerader** [comment]
> Screenshot 2024-11-21 at 10 50 02 AM
> 
> it is partially missing. is this an expected behavior?

@koji ya so the animations get cropped in order to fit the space. If you adjust the viewport, it will not crop that part out.

**Comment by koji** [comment]
From the description, you did optimize webm files already.
Do you remember how much size the optimization reduced the original file size?
maybe using ffmpeg to optimize webm files.

**Review by koji — APPROVED** [review]
at this moment I don't have any specific idea.

**Review by koji — COMMENTED** [review]
need some changes for OT-2

**Pull request merged**